Sources/UAPusher/UAManager.swift

```diff
@@ -31,6 +31,8 @@ public final class UAManager {
     }
     
     public func send(request: UARequest) throws -> UAResponse {
+        // For debugging
+        try print(request.getBody().makeResponse())
```

Comment:
I think this should be a config setting or left out. Similar to `fluent.json` where you can set `log: true`

Reply:
Removing for now... Creating an issue for the config-idea